hadoop-client-modules/hadoop-client-runtime/pom.xml

```diff
@@ -229,6 +229,13 @@
                         <exclude>update*</exclude>
                       </excludes>
                     </filter>
+                    <filter>
+                      <artifact>com.google.protobuf:protobuf-java</artifact>
+                      <excludes>
+                        <exclude>google/protobuf/*.proto</exclude>
+                        <exclude>google/protobuf/**/*.proto</exclude>
+                      </excludes>
+                    </filter>
```

Comment:
These classes are shaded or they are excluded from shading?

Reply:
proto files are excluded from shading. 
.5.0 protobuf jar didnt have proto files in jars. 3.7.1 jar jar have proto files, so shaded jar verifier was complaining about unwanted content in shaded jars.